website/docs/r/workloads_sap_discovery_virtual_instance.html.markdown

```diff
@@ -0,0 +1,235 @@
+---
+subcategory: "Workloads"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_workloads_sap_discovery_virtual_instance"
+description: |-
+  Manages a SAP Discovery Virtual Instance.
+---
+
+# azurerm_workloads_sap_discovery_virtual_instance
+
+Manages a SAP Discovery Virtual Instance.
+
+-> **Note:** Before using this resource, it's required to submit the request of registering the Resource Provider with Azure CLI `az provider register --namespace "Microsoft.Workloads"`. The Resource Provider can take a while to register, you can check the status by running `az provider show --namespace "Microsoft.Workloads" --query "registrationState"`. Once this outputs "Registered" the Resource Provider is available for use.
+
+## Example Usage
+
+```hcl
+data "azurerm_subscription" "current" {}
+
+resource "tls_private_key" "example" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+data "tls_public_key" "example" {
+  private_key_pem = tls_private_key.example.private_key_pem
+}
```

Comment:
shall we remove 3rd party resources like `tls_private_key` and `tls_public_key`?

Reply:
Updated